### ISSUE TITLE
Add column to users table ... e.g. sendThankYou

### ISSUE THREAD

**Issue by SantoshSrinivas79** [yogiben/meteor-admin#346]
Is it possible to add a button to the "view all" table of "users" with an additional column containing a button like "sendThankyou" which can be made to trigger a thank you email using a method.

**Comment by challett** [comment]
Yes, inside of your tableColumns array add something like:                
`{ label: 'Thank', name: 'thank', template: 'sendThankYou'}`.  From here you will need to define a template named `sendThankYou` which would contain only a button.  You can then define event handlers on this template to execute any logic you want.

**Comment by MichaelJCole** [comment]
@challett how to you access the tableColumns array for users?

I'm getting this error: `Error: Users is not in the [object global]` on this code:

```
AdminConfig = {
  collections: {
    Users: {
      icon: 'user',
      omitFields: ['updatedAt', 'createdAt', 'username'],
    },
  },
};
```

**Comment by MichaelJCole** [comment]
https://github.com/yogiben/meteor-admin/issues/319

**Comment by challett** [comment]
I'm not currently using the user collection part of this plugin so I am not sure,  sorry.

**Comment by challett** [comment]
In retrospect maybe that's why this issue came about. I didn't think the user collection would act any different to the other ones.
